public/locales/ca-CA/translations.json

```diff
@@ -522,5 +524,11 @@
   "note": "Nota",
   "indicate_unl": "indica un validador en una UNL",
   "transaction_tokens_involved": "<Currency/> i <Currency2/>",
-  "transaction_tokens_swapped": "<Currency/> per <Currency2/>"
+  "transaction_tokens_swapped": "<Currency/> per <Currency2/>",
```

Comment:
```suggestion
  "transaction_tokens_swapped": null,
```

Reply:
Not sure why this is null since this key is not related to this PR

Reply:
Oh oops, misread the diff